Trino Hook: Add ability to read JWT from file (#31950)
mistercrunch/Airflow: docs/apache-airflow-providers-trino/connections.rst

    The following extra parameters can be used to configure authentication:

    * ``jwt__token`` - If jwt authentication should be used, the value of token is given via this parameter.

>     * ``jwt__file``  - If jwt authentication should be used, the location on disk for the token is given with this parameter.

Reviewer: ```suggestion
    * ``jwt__file``  - If jwt authentication should be used, the location on disk for the file containing the jwt token.
```
